test/framework/easyblock.py

```diff
@@ -407,6 +407,31 @@ def test_get_easyblock_instance(self):
         logtxt = read_file(eb.logfile)
         self.assertTrue(eb_log_msg_re.search(logtxt), "Pattern '%s' found in: %s" % (eb_log_msg_re.pattern, logtxt))
 
+    def test_patchlevel(self):
+        """Test the parsing of the fetch_patches function."""
+        # adjust PYTHONPATH such that test easyblocks are found
+        testdir = os.path.abspath(os.path.dirname(__file__))
+        ec = process_easyconfig(os.path.join(testdir, 'easyconfigs', 'toy-0.0-patches.eb'))[0]
+        eb = get_easyblock_instance(ec)
+
+        patches = [
+            ('toy-0.0_level0_2.patch',0),  # should also be level 0 (not None)
+            ('toy-0.0_level4.patch',4),   # should be level4
+        ]
+        sandbox_sources = os.path.join(testdir, 'sandbox', 'sources')
+        init_config(args=["--sourcepath=%s" % sandbox_sources])
```

Comment:
this is already done via `setUp` in `EnhancedTestCase`, so remove these two lines